drivers/media/i2c/imx290.c

```diff
@@ -312,6 +358,11 @@ static const struct imx290_mode imx290_modes_2lanes[] = {
 		.link_freq_index = FREQ_INDEX_1080P,
 		.data = imx290_1080p_settings,
 		.data_size = ARRAY_SIZE(imx290_1080p_settings),
+		.clk_data = {
+			imx290_37_125mhz_clock_1080p,
```

Comment:
`[CLK_37_125] = ...`? Or a comment about the order? I'll take a no.

Reply:
There is a comment in the definition of struct imx290_mode
```
	/* Clock setup can vary. Index as enum imx290_clk_index */
	const struct imx290_regval *clk_data[2];
```
but it doesn't hurt to be explicit.
Done.